lighthouse-core/test/test-utils.js

```diff
@@ -38,7 +38,7 @@ expect.extend({
     // done for asymmetric matchers anyways.
     const thisObj = (this && this.utils) ? this :
         {isNot: false, promise: ''};
-
+    // @ts-ignore
```

Comment:
The error is inscrutable and I don't even know what to say about it. tried to explicitly list the args instead of use spread but no luck.

![image](https://user-images.githubusercontent.com/4071474/94846575-f86a2780-03e6-11eb-8530-84b4039f02f4.png)